src/generic.rs

```diff
@@ -67,19 +71,8 @@ where
     T: Beef + ?Sized,
     U: Capacity,
 {
-    /// Borrowed data.
-    ///
-    /// # Example
-    ///
-    /// ```rust
-    /// use beef::Cow;
-    ///
-    /// let borrowed: Cow<str> = Cow::borrowed("I'm just a borrow");
-    /// ```
-    #[inline]
-    pub fn borrowed(val: &'a T) -> Self {
-        let (ptr, fat, cap) = T::ref_into_parts::<U>(val);
-
+    // fixme: it should be unsafe :)
+    const fn from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
```

Comment:
```suggestion
    const unsafe fn from_raw_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
```

Obviously the change needs to be propagated. Even though field access is safe and it *is* formally acceptable to have non-exported safe functions with safety requirements, it's much better practice to mark them `unsafe`.